### DISCUSSION TITLE
Are there any plans for the code / executables to be signed?

### DISCUSSION THREAD

**Discussion by colinbridle** [MscrmTools/XrmToolBox#1006] [Q&A]
Hi,

Looking at using this tool, but the team looking after the managed environment I want to install the tool on to have told me the executable is not digitally signed and the publisher is being blocked by Windows smart screen.  

So is  that correct that the executable / code has not been signed?  And is there a reason that Windows Smart screen should be blocking?

Thanks

**Comment by MscrmTools** [comment]
It seems I forgot to sign the tool with the latest version.
I have a new version planned anyway. I will sign it

**Reply by colinbridle** [reply]
Thanks

**Reply by MscrmTools** [reply]
New version released which is signed
